edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/ClientTokenCloudConnection.cs

```diff
@@ -115,7 +117,7 @@ public async Task<ICloudProxy> UpdateTokenAsync(ITokenCredentials newTokenCreden
                                 // complete the tokenGetter
                                 if (this.tokenGetter.HasValue)
                                 {
-                                    if (TokenHelper.IsTokenExpired(this.Identity.IotHubHostName, newTokenCredentials.Token))
+                                    if (TokenHelper.IsTokenExpired(this.Identity.IotHubHostname, newTokenCredentials.Token))
```

Comment:
We should continue to use IoT hub hostname for any token related calls, right?

Reply:
For this call, this is correct.